**Монету подбросили 144 раза. Какова вероятность, что орел выпадет ровно 70 раз?**

Воспользуемся формулой **Бернулли**:

$P_{n}(X=k)=C^k_{n}p^{k}q^{n-k}$, где 

                                        n=144 - количество испытаний,   
                                        k=70 - количество выпадения орла,  
                                        p=0.5 - вероятность выпадения орла,  
                                        q=0.5 - вероятность выпадения решки

Создадим функцию расчета по формуле Бернулли:

In [2]:
from math import factorial, exp
def bernulli(n, k, p):
    comb=factorial(n)/(factorial(k)*factorial(n-k))
    return comb*(p**k)*(1-p)**(n-k)

и вычислим результат:

In [3]:
print(f'P = {bernulli(144,70,0.5): .4f}')

P =  0.0628


___  
_Ответ: вероятность того, что при 144-х кратном подбрасывании монетки орел выпадет ровно 70 раз составляет **P=0.0628** или **$\approx$ 6.28%**_
___